src/state/actions/addQuestionAction.js

```diff
@@ -0,0 +1,15 @@
+import { ADD_QUESTION_TYPE } from '../../constants';
+
+const addQuestion = (id, title, quizId, answers) => {
+  return {
+    type: ADD_QUESTION_TYPE,
+    payload: {
+      id: id,
+      title: title,
+      quiz: quizId, // quiz id goes here
+      answers: [...answers],
+    },
+  };
+};
+
+export default addQuestion;
```

Comment:
That's action's file, it is expected that here will be more that one action, it'll be better if you use `export const ` instead of default one